spec/config/initializers/date_formats_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'Time::DATE_FORMATS' do
+  describe ':govuk_date_and_time, :govuk_time formats' do
+    it 'formats time to indicate midday' do
+      date_and_time = Time.zone.local(2020, 12, 25, 12, 0, 0)
```

Comment:
Following on from my other comment, I'd prefer this to say

```suggestion
      date_and_time = Time.zone.local(2020, 12, 25, 12, 0, 3)
```

and see that the output still says '12pm (midday)'.